packaging/language/easy_install.py

```diff
@@ -70,6 +70,12 @@
     version_added: "1.3"
     required: false
     default: null
+  state:
+    description:
+      - The desired state of the gem. C(latest) ensures that the latest version is installed.
+    required: false
+    choices: [present, latest]
+    default: present
```

Comment:
Also add a version_added: "2.0" field so people know which version of ansible this becomes available in.